doc/source/style-actions/index.rst

```diff
@@ -46,6 +46,27 @@ Doc style action
     {% endfor %}
 
 
+Docker style action
+-------------------
+.. jinja:: docker-style
```

Comment:
```suggestion
-------------------

.. jinja:: docker-style
```

Reply:
I'll apply this to `code-style` and `doc-style`